frontend/src/views/ArticleView.vue

```diff
@@ -4,15 +4,26 @@ import InputText from "primevue/inputtext";
 import { ref } from "vue";
 import { onMounted } from "vue";
 
-import { getArticles } from "@/backendClient";
+import { createArticle, getArticles } from "@/backendClient";
 import ArticlesItem from "@/components/articles/ArticleItem.vue";
-//
+import { ArticleKind } from "@/global/types/appTypes";
 import type { Article } from "@/global/types/entities";
 
 async function reloadPage() {
   articlesList.value = (await getArticles()).data;
 }
 const articlesList = ref<Article[]>([]);
+
+// temp solution
```

Comment:
I would suggest such prefixes to be able to find them easily later.

```suggestion
// TODO: temp solution
```